.github/workflows/ci.yml

```diff
@@ -29,19 +30,45 @@ jobs:
               - .github/workflows/ci.yml
               - codecov.yml
               - setup.cfg
+              - Makefile
               - pyproject.toml
+              - setup.py
               - scripts/ci/config.yml
               - scripts/ci/run_tests.sh
               - requirements/tests-requirements.txt
+              - "hooks/**"
             bentoml:
               - *related
               - "bentoml/**"
               - "tests/**"
-              - "docker/**"
             docs:
               - *related
               - requirements/docs-requirements.txt
               - "docs/**"
+            docker_readme:
+              - docker/generated/bento-server/README.md
+
+  bento_server_push_readme:
```

Comment:
could we add a CI task on new commits on main branch, that builds and pushes new devel images?

Reply:
Let's do a nightly deploy? Probably best not to have *too* many releases on docker hub :P